packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownVariables.ts

```diff
@@ -85,9 +85,9 @@ export const dropdownVariables = (siteVars): DropdownVariables => ({
   belowListBorderRadius: `0 0 ${siteVars.borderRadiusMedium} ${siteVars.borderRadiusMedium}`,
   listBackgroundColor: siteVars.colorScheme.default.background,
   listBorderColor: 'transparent',
-  listBorderWidth: '0px',
+  listBorderWidth: '0.1rem',
```

Comment:
other attributes are using pxToRem, why is this one using `rem`?

Reply:
Thanks for pointing it out, changed it to pxToRem.